monai/transforms/utility/array.py

```diff
@@ -316,6 +319,42 @@ def __call__(self, img) -> torch.Tensor:
         return torch.as_tensor(img)
 
 
+class EnsureTensor(Transform):
+    """
+    Ensure the input data to be a PyTorch Tensor, support: `numpy array`, `PyTorch Tensor`, `float`, `int`, `bool`.
+    If passing a dictionary, list or tuple, recursively check every item and ensure it to be PyTorch Tensors.
+
+    """
+
+    def __call__(self, data):
+        """
+        Args:
+            data: input data can be PyTorch Tensor, numpy array, list, dictionary, int, float, bool, str, etc.
+                will ensure Tensor, Numpy array, float, int, bool as Tensors, strings and objects keep the original.
+                for dictionay, list or tuple, ensure every item as a Tensor if applicable.
+
+        """
+        if isinstance(data, torch.Tensor):
```

Comment:
could you create utility functions for this and the inverse? then the transforms could use the utils

Reply:
Hi @wyli ,

Sure, Thanks for your review.
I updated to utils in the latest commit.
Could you please help review it again?

Thanks in advance.